libp2p/Session.cpp

```diff
@@ -22,18 +22,26 @@ Session::Session(Host* _h, unique_ptr<RLPXFrameCoder>&& _io, std::shared_ptr<RLP
     m_socket(_s),
     m_peer(_n),
     m_info(_info),
-    m_ping(chrono::steady_clock::time_point::max()),
-    m_logContext(_info.id.abridged() + "|" + _info.clientVersion)
+    m_ping(chrono::steady_clock::time_point::max())
 {
+    stringstream remoteInfoStream;
+    remoteInfoStream << "(" << m_info.id << "@" << m_socket->remoteEndpoint() << ")";
```

Comment:
abridged?

Reply:
Yes it's abridged: 

`TRACE 05-23 20:30:25 p2p  net    Pong to (##44eed8b3…@35.195.118.171:30303)`